data/help.txt

```diff
@@ -14,7 +14,7 @@ help "bank"
 
 help "bank advanced"
 	`When entering a credit value at the bank, you can put in an exact number, or use the suffixes K, M, B, and T as shorthand for thousands, millions, billions, and trillions. For example, entering a value such as "100k" would count as 100,000 credits.`
-	`Entering a number that is higher than what you or the bank have available depending if you are paying or applying for a loan will default to the maximum available value. For example, entering "1m" to pay off a 100,000 credit loan will completely pay off the loan without giving extra, should you have available credits.`
+	`Entering a number that is higher than what you or the bank have available will default to the maximum available value. For example, entering "1m" to pay off a 100,000 credit loan will completely pay off the loan without giving extra, should you have available credits.`
```

Comment:
Not a typo.
```suggestion
	`Entering a number that is higher than what you or the bank have available depending if you are paying or applying for a loan will default to the maximum available value. For example, entering "1m" to pay off a 100,000 credit loan will completely pay off the loan without giving extra, should you have available credits.`
```

Reply:
I'm pretty sure that 'depending if' is not a correct from here. 'Depending on whether' or something like that would work better.

Reply:
"Depending on if" would be the simplest typo fix.

Reply:
There's nothing wrong with "depending if." Any changes to this would be matters of preference.

Reply:
>  "Depending on if" would be the simplest typo fix.

After prepositions, 'whether' is preferred over 'if':https://dictionary.cambridge.org/us/grammar/british-grammar/if-or-whether

> here's nothing wrong with "depending if." 

Depending needs a preposition. 'if' is **not** a preposition. 
https://lingohelp.me/preposition-after-adjective/depending-upon-or-depending-of/
https://www.espressoenglish.net/when-to-use-depend-vs-depending/